event_routing_backends/processors/xapi/tests/fixtures/expected/edx.grades.course.grade_calculated.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "32e08e30-f8ae-4ce2-94a8-c2bfe38a70cb",
+  "result": {
+    "score": {
+      "scaled": 0.02,
+      "raw": 0.02,
+      "min": 0.0,
+      "max": 1.0
+    },
+    "extensions": {}
```

Comment:
I think this should just be omitted if it's empty?

Reply:
It was fixed by setting a default value for empty `letter_grade`.